mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/CassandraMailboxSessionMapperFactory.java

```diff
@@ -61,9 +62,32 @@
 
 /**
  * Cassandra implementation of {@link MailboxSessionMapperFactory}
- * 
  */
 public class CassandraMailboxSessionMapperFactory extends MailboxSessionMapperFactory {
+
+    public static CassandraMailboxSessionMapperFactory forTests(Session session, CassandraTypesProvider typesProvider,
```

Comment:
if it's for test, then it must be in a test class

Reply:
DONE